packages/apps/public/locales/it/translation.json

```diff
@@ -1348,16 +1348,16 @@
   "{{currency}} total": "{{currency}} totale",
   "{{days}} days": "{{days}} giorni",
   "{{done}}/{{total}}, {{progress}}% done": "{{done}}/{{total}}, {{progress}}% done",
-  "{{d}} days": "{{d}} giorni",
+  "{{days}} days": "{{days}} giorni",
```

Comment:
```suggestion
```

Removing, duplicated 2 lines above (as said earlier, change in i18n keys to make it work as expected)